Frontend/research/src/App.css

```diff
@@ -67,19 +160,42 @@ header nav{
   margin-bottom: 10px;
 }
 
+nav.header div{
+  display: inline-flex;
+  width: 100%;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
 nav a{
   color: white;
-  font-size: 30px;
-  padding: 20px;
-  border-radius: 10px 10px 0 0 ;
+  padding: 10px;
+  border-radius: 5px 5px 0 0;
   font-size: .8em;
   text-decoration: none;
 }
 
+.header{
+  padding:12px;
+  height: auto;
+}
+
+.home{
+  background-color: rgba(0, 0, 0, 0.664);
+  width: 35%;
+  margin: 0 auto;
+  padding: 20px;
+  margin-top: 40px;
+  color: white;
+  border-radius: 100px;
+  border: 1px solid white;
+}
+
 
 nav a:hover{
   background-color: white;
   color: black;
+  transition: .5s;
```

Comment:
was the transition working here? if not, try adding that to the nav a styles and then the changes you want in the hover styles... so in the styles for nav a it will look like this 
nav a {
  color: white;
  padding: 10px;
  border-radius: 5px 5px 0 0;
  font-size: .8em;
  text-decoration: none;
  transition: all .5s ease-in
}
i also like to add in an ease property there are many available and you can add custom easing as well. definitely worth a search on custom transition easing